### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by amon-cofie**
### Changes:

- Created a table named vets with the following columns:

> id: integer (set it as autoincremented PRIMARY KEY)
> name: string
> age: integer
> date_of_graduation: date

-  Created a "join table" called specializations to handle the many-to-many relationship between the tables species and vets
-  Created a "join table" called visits to handle the many-to-many relationship between the tables animals and vets
- Inserted the following data for vets:

> Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
> Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
> Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
> Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.

- Inserted the following data for specialties:

> Vet William Tatcher is specialized in Pokemon.
> Vet Stephanie Mendez is specialized in Digimon and Pokemon.
> Vet Jack Harkness is specialized in Digimon.

- Inserted the following data for visits:

> Agumon visited William Tatcher on May 24th, 2020.
> Agumon visited Stephanie Mendez on Jul 22th, 2020.
> Gabumon visited Jack Harkness on Feb 2nd, 2021.
> Pikachu visited Maisy Smith on Jan 5th, 2020.
> Pikachu visited Maisy Smith on Mar 8th, 2020.
> Pikachu visited Maisy Smith on May 14th, 2020.
> Devimon visited Stephanie Mendez on May 4th, 2021.
> Charmander visited Jack Harkness on Feb 24th, 2021.
> Plantmon visited Maisy Smith on Dec 21st, 2019.
> Plantmon visited William Tatcher on Aug 10th, 2020.
> Plantmon visited Maisy Smith on Apr 7th, 2021.
> Squirtle visited Stephanie Mendez on Sep 29th, 2019.
> Angemon visited Jack Harkness on Oct 3rd, 2020.
> Angemon visited Jack Harkness on Nov 4th, 2020.
> Boarmon visited Maisy Smith on Jan 24th, 2019.
> Boarmon visited Maisy Smith on May 15th, 2019.
> Boarmon visited Maisy Smith on Feb 27th, 2020.
> Boarmon visited Maisy Smith on Aug 3rd, 2020.
> Blossom visited Stephanie Mendez on May 24th, 2020.
> Blossom visited William Tatcher on Jan 11th, 2021.

- Wrote queries to answer the following:

> Who was the last animal seen by William Tatcher?
> How many different animals did Stephanie Mendez see?
> List all vets and their specialties, including vets with no specialties.
> List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
> What animal has the most visits to vets?
> Who was Maisy Smith's first visit?
> Details for most recent visit: animal information, vet information, and date of visit.
> How many visits were with a vet that did not specialize in that animal's species?
> What specialty should Maisy Smith consider getting? Look for the species she gets the most.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -73,4 +73,35 @@ WHERE name IN ('Charmander','Squirtle', 'Blossom');
...
+(5,4,'2021-02-24'),
+(6,3,'2019-09-29'),
+(7,4,'2020-10-03'),
+(7,4,'2020-11-04'),
+(8,2,'2019-01-24'),
+(8,2,'2019-05-15'),
+(8,2,'2020-02-27'),
+(8,2,'2020-08-03'),
+(9,3,'2020-05-24'),
+(9,1,'2021-01-11');
```
Comment (thecodechaser) [inline data.sql]: - [ ] You missed adding the following data to `visits`, Please make sure to add them.
  - Plantmon visited Maisy Smith on Dec 21st, 2019.
  - Plantmon visited William Tatcher on Aug 10th, 2020.
  - Plantmon visited Maisy Smith on Apr 7th, 2021.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @amon-cofie👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (amon-cofie) [reply data.sql]: Thanks a lot for pointing this out. I will make the correction immediately

**Review by omarsalem7 — APPROVED** [review]
Hi @amon-cofie 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Implement the changes requested by the reviewer
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**